models/turbine_models/tests/sdxl_test.py

```diff
@@ -502,98 +400,103 @@ def test04_ExportVaeModelEncode(self):
         np.testing.assert_allclose(torch_output, turbine, rtol, atol)
 
     def test05_t2i_generate_images(self):
```

Comment:
seems like this test is failing the CI with this error:
`ValueError: Expected input 4 to be of shape (2, 6) for compiled_unet['run_forward'], got (1, 6).`